docs/bare-metal/README.md

```diff
@@ -0,0 +1,298 @@
+        172.16.222.16 edge01
+        172.16.222.17 edge02
+
+Copy the /etc/host file over.
+
+    ansible all -i inventory -u centos --sudo --ask-sudo-pass -m copy -a "src=hosts dest=/etc/hosts"
+
+
+Might as well set the timezone just for grins.
+
+    ansible all -i inventory -u centos --sudo --ask-sudo-pass -m command -a "timedatectl set-timezone America/Los_Angeles"
+
+
+## Run It!
+
+You now are ready to run the playbook.  Change directory to the mantl root.  Your inventory should be there as well as
+the file `security_setup` (along with quite a few others).
+
+Run the security-setup script:
+
+    ./security-setup
+
+It asks for one admin password. At the end of that run there will be a `security.yml` file.  It will have the password
+you entered and a lot of keys and such that are used in the installs and in the installed software.
+
+The file you will be running is `terraform.sample.yml`.  Since you created your own inventory and didn't use terraform,
+there are a few environment variables you need to set for your run.
+
+    ansible-playbook -u centos -i inventory \
+            -e provider=bare-metal \
+            -e consul_dc=dc1 \
+            -e docker-lvm-backed=true \
+            -e docker_lvm_data_volume_size="80%FREE" \
+            -e @security.yml  terraform.sample.yml >& bare-metal.log
+
+
+In another window tail -f  that log file to follow whats going on.
+
+The meaning of the parts of this command are as follows:
+
+ * ansible-playbook -u centos -i inventory
+   * run the ansible play book as centos user against the inventory found in the ./inventory file.
+ * -e provider=bare-metal
+   * The "provider" is not terraform for a cloud, but rather bare-metal. If the inventory had been generated
+   by terraform on Google Cloud, this value would have been set automatically to 'gcs'
+ * -e consul_dc=dc1
+   * This is the name found in your ./inventory file for your datacenter.
+ * -e docker-lvm-backed=true
+   * lvm back docker is a really good idea in centos. This is why you craeted the extra partion during installation.
+ * -e docker_lvm_data_volume_size="80%FREE"
+   * This defaults to "40%FREE" in the docker role because the default LVM partition is shared with other things.
+   You could leave this off, but its likely with your own hardware you will have different constraints and its a good
+   variable to know.
+ * -e @security.yml
+   * this a series of environment variable settings that have all the security settings of the varios parts of mantl. The @
+   causes ansible to evaluate the file.
+ * terraform.sample.yml
+   * this is the ansible file that is being run.
+ * >& bare-metal.log
+   * this redirects the output to a file so that you can review it later.   Tailing with a -f flag lets you watch the
+   progress as ansible works through the rolls accross your inventory.
+
+
+Once you are done go to the browser and go to the /ui directory of any control node and you should see the mantlui.
```

Comment:
Maybe provide an example based on the IPs above? Like "navigate to the IP of one of your control nodes, e.g. 172.16.222.6".